pages/learn/deploy/release-strategy/update-locking.md

```diff
@@ -80,6 +80,6 @@ Check the link for more examples and other Python libraries that provide locking
 
 The update lock can be overridden in case you need to force an update, for instance, if your app has hung in a critical section.
 
-The way to do this is hitting the `/v1/update` endpoint of the [supervisor HTTP API](./API.md), with `{ "force": true }` as body.
+The way to do this is hitting the `/v1/update` endpoint of the [supervisor HTTP api](/reference/supervisor/supervisor-api/#post-v1update), with `{ "force": true }` as body.
```

Comment:
```suggestion
The way to do this is hitting the `/v1/update` endpoint of the [supervisor HTTP API](/reference/supervisor/supervisor-api/#post-v1update), with `{ "force": true }` as body.
```

I think `API` should be all caps?

Reply:
You are absolutely right. I dunno how that got switched.